.github/workflows/ci.yml

```diff
@@ -462,6 +466,9 @@ jobs:
     steps:
       - name: "Install Git and checkout project"
         uses: actions/checkout@v4
+        with:
+          repository: ${{ github.event.pull_request.head.repo.full_name }}
+          ref: ${{ github.event.pull_request.head.ref }}

```

Comment:
Why do you need this? Isn't this done by default already?
```suggestion
```

Reply:
Because of this: 

https://github.com/EndBug/add-and-commit#working-with-prs

:)